mcs_benchmark_data/_benchmark_submission_transformer.py

```diff
@@ -0,0 +1,121 @@
+import json
+import os
+from time import strptime
+from pathlib import Path
+from typing import Generator
+from rdflib import URIRef
+
+from mcs_benchmark_data._model import _Model
+from mcs_benchmark_data._transformer import _Transformer
+from mcs_benchmark_data.models.submission import Submission
+from mcs_benchmark_data.models.submission_sample import SubmissionSample
+from mcs_benchmark_data.models.test_score import TestScore
+from mcs_benchmark_data.models.dev_score import DevScore
+
+
+class _BenchmarkSubmissionTransformer(_Transformer):
+
+    """
+    Abstract base class for benchmark transformers.

```

Comment:
Benchmark submissions?

Reply:
I assume you're not using this class yet, probably should have waited for another PR, leave it for now.